federation/pkg/federation-controller/replicaset/replicasetcontroller.go

```diff
@@ -589,7 +566,6 @@ func (frsc *ReplicaSetController) reconcileReplicaSet(key string) (reconciliatio
 			"Replicaset update in cluster %s failed: %v", op.ClusterName, operror)
 	})
 	if err != nil {
-		glog.Errorf("Failed to execute updates for %s: %v", key, err)
```

Comment:
There are lots of glog.Errorfs in this func before returning statusError.
You want to remove all of them?